frontend/src/v5/ui/routes/dashboard/projects/containers/containersList/containersList.styles.ts

```diff
@@ -14,8 +14,31 @@
  *  You should have received a copy of the GNU Affero General Public License
  *  along with this program.  If not, see <http://www.gnu.org/licenses/>.
  */
+
 import styled from 'styled-components';
+import * as SearchInputStyles from '@controls/searchInput/searchInput.styles';
 
 export const Container = styled.div`
 	margin: 16px 0;
 `;
+
+export const CollapseSideElementGroup = styled.div`
+	display: flex;
+	align-items: center;
+
+	${SearchInputStyles.TextField} {
+		margin-right: 15px;
+		
+		@media (max-width: 768px) {
+			width: 235px;
+		}
+		
+		@media (max-width: 1024px) {
+			max-width: 405px;
+		}
+		
+		@media (min-width: 1025px) {
```

Comment:
There's a difference of one pixel in these two media queries. 
Was this a copy paste error?
Also we should use constants